### PR TITLE
CSS + Parallel Docker Web

### LINKED ISSUE #569: Make Docker web parallelized (default to all tabs)
Also change it to use just the simple tab name for the container name for easier docker logs and docker kill

### LINKED ISSUE #631: Replace generic groupBy/sortBy in Components/TopbarComponent with lodash

### PR DESCRIPTION
**Pull request by adrw**
* All Blueprint CSS now included
* CSS is minified to output `styles.css` instead of `styles.js` for 16% size savings
* [CLOSES #569] `gradle/web.gradle` now by default spins up parallel docker instances, tags output, and also sends to web/logs directory
* Improved Docker documentation which includes which `@misk/` packages shipped in which docker image
* `@misk/common` and `@misk/components` prepare tasks updated to use Gradle Docker builds when publishing to NPM instead of relying on local builds
* `misk-web@0.0.5`
* [CLOSES #631] Replace sortBy and groupBy with lodash equivalents

### CODE REVIEW

File: misk/web/@misk/common/README.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ $ yarn add @misk/common
     ```HTML
     <body>
 
+      <link rel="stylesheet" type="text/styles"
```
Comment (jjestrel) [inline misk/web/@misk/common/README.md]: Does this need a />

File: misk/web/@misk/components/package.json [hunk 2]
```diff
@@ -23,16 +23,16 @@
     "ci-build": "yarn run clean && yarn run lib",
     "lib": "webpack",
     "lint": "prettier --write --config prettier.config.js \"./src/**/*.{md,css,sass,less,json,js,jsx,ts,tsx}\"",
-    "prepare": "yarn run lint && yarn run test && yarn run lib",
+    "prepare": "cd ../../../../ && pwd && ./gradlew web -Pcmd='-a' -Ptabs='@misk/components'",
```
Comment (jjestrel) [inline misk/web/@misk/components/package.json]: Does this need a PWD?

File: docker/misk-web/Dockerfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 FROM node:alpine
 
-RUN apk add --no-cache tini
+RUN apk add --no-cache tini bash sed
```
Comment (jjestrel) [inline docker/misk-web/Dockerfile]: Does this need `bash`?

File: misk/web/@misk/common/README.md (hunk 1 above)
Comment (adrw) [reply misk/web/@misk/common/README.md]: Thanks for catching that!

**Pull request merged**